actions/techdocs-rewrite-relative-links/action.yaml

```diff
@@ -38,13 +38,13 @@ runs:
 
   steps:
     - name: Checkout
-      if: ${{ !inputs.skip-checkout }}
+      if: ${{ inputs.checkout }}
       env:
         # In a composite action, these two need to be indirected via the
         # environment, as per the GitHub actions documentation:
         # https://docs.github.com/en/actions/learn-github-actions/contexts
-        action_repo: ${{ github.action_repository }}
-        action_ref: ${{ github.action_ref }}
+        action_repo: ${{ github.action_repository && github.action_repository || 'grafana/shared-workflows' }}
```

Comment:
Could you please explain this to me? Would it be different than `${{ github.action_repository || 'grafana/shared-workflows' }}`?
I guess we would need this expression in case `grafana/shared-workflows` was also a context value where we didn't know if it's not null/empty? But now since it's hardcoded I guess it's the same? Unless I miss something obvious

Reply:
That part is currently experimental. For some reason the conditional checkout does not work (it's always checked out) and I ran into issues with `github.action_repository`. This whole thing is just weird right now and I'm still trying to find out what is going on here :(

Reply:
Looks like "action_repository" etc. are only available when you don't run the action as local action. We might get somewhere with `workflow_ref` in this scenario, but that also does not work in the pull-request context 🙁